docs/stransmit.md

```diff
@@ -62,10 +62,29 @@ and you should see bars and tone right away.
 If you're having trouble, make sure this works, then add complexity one step at a time (multicast, push vs listen, etc)
 
 
+General medium specification
+----------------------------
+
+The media are specified as the standard URI format:
+
+```
+    SCHEME://HOST:PORT?PARAM1=VALUE&PARAM2=VALUE...
+```
+
+This application handles the following schemes:
+
+* `file` - for file or standard input and output
+* `udp` - UDP output (unicast and multicast)
+* `srt` - SRT connection
+
+(Note that this application doesn't support file as a medium, but this
+can be handled by other applications from this project).
+
+
 Medium: FILE (including standard process pipes)
 -----------------------------------------------
 
-**NB!** File mode, except `file://con` is supported in *srt-file-transmit* tool!
+**NB!** File mode, except `file://con`, is not supported in *srt-file-transmit* tool!
```

Comment:
**NB!** File mode, except `file://con`, is not supported in the *srt-file-transmit* tool!